ddp-workspace/projects/ddp-sdk/src/lib/ddp.module.ts

```diff
@@ -238,7 +237,6 @@ export function createTranslateLoader(http: HttpClient): TranslateHttpLoader {
 @NgModule({
     imports: [
         CommonModule,
-        BrowserAnimationsModule,
```

Comment:
Removed from shared `ddp-sdk` module. 
Move import of `BrowserAnimationsModule` to `app.module` of every single study.